internal/service/ipam/api_range_data_source_test.go

```diff
@@ -0,0 +1,123 @@
+package ipam_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/infobloxopen/bloxone-go-client/ipam"
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/acctest"
+)
+
+func TestAccRangeDataSource_Filters(t *testing.T) {
+	dataSourceName := "data.bloxone_ipam_ranges.test"
+	resourceName := "bloxone_ipam_range.test"
+	var v ipam.IpamsvcRange
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		CheckDestroy:             testAccCheckRangeDestroy(context.Background(), &v),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRangeDataSourceConfigFilters("10.0.0.20", "10.0.0.8"),
```

Comment:
nit:, but the ordering of parameters could have been `start,end` instead of `end,start` :D I think the generator is to be blamed?

Reply:
Yes, the generated code. I kept it as is.

Reply:
Fixed the generated code.

Reply:
Thanks.